src/sequence_batch.h

```diff
@@ -81,6 +81,25 @@ class SequenceBatch {
     return negative_sequence_batch_[sequence_index];
   }
 
+  // big_endian: N_pos is in the order of sequence
+  // little_endian: N_pos is in the order from the sequence right side to left, 
+  //                this is the order of the GenerateSeed
```

Comment:
Didn't get this comment. Can you clarify more?

Reply:
I've updated the comments for this part.